tests/adapters/test_serial.py

```diff
@@ -64,6 +64,17 @@ def test_read_bytes(adapter):
     assert adapter.read_bytes(11) == b"basd\x02fasdf\n"
 
 
+def test_read_bytes_unlimited(adapter):
+    adapter.write_bytes(b"basd\x02fasdf\n")
```

Comment:
I guess that should have a `\n` (and associated `read_termination` setting) in the middle of the string (like the next test) to confirm that the default indeed is not stopped by a termchar?

Reply:
Basically I have a test for reading the whole string, neglecting the Termchar in middle option, and tests for either option, but yes, we can have one test for the default (testing the not splitting together with just reading all bytes) and one for the option.